### PR TITLE
Added function GetIssuerByLabels

### PR DESCRIPTION
**Pull request by vakwetu**

### CODE REVIEW

**Review by olliewalsh — APPROVED** [review]
/lgtm

File: modules/certmanager/test/functional/certmanager_test.go [hunk 1]
```diff
@@ -50,6 +50,16 @@ var _ = Describe("certmanager module", func() {
 			g.Expect(issuer.Spec.SelfSigned).NotTo(BeNil())
 			g.Expect(issuer.ObjectMeta.Labels["f"]).To(Equal("l"))
 		}, timeout, interval).Should(Succeed())
+		Eventually(func(g Gomega) {
```
Comment (stuggi) [inline modules/certmanager/test/functional/certmanager_test.go:53]: could also add tests for the `==0` and `>1`  case. we can do that in a follow up

**Review by stuggi — APPROVED** [review]
/lgtm

File: modules/certmanager/test/functional/certmanager_test.go (hunk 1 above)
Comment (stuggi) [reply modules/certmanager/test/functional/certmanager_test.go:53]: https://github.com/openstack-k8s-operators/lib-common/pull/468

**Pull request merged**